lucene/core/src/java/org/apache/lucene/store/Directory.java

```diff
@@ -165,4 +165,13 @@ public void copyFrom(Directory from, String src, String dest, IOContext context)
    * @throws AlreadyClosedException if this Directory is closed
    */
   protected void ensureOpen() throws AlreadyClosedException {}
+  
```

Comment:
Now you mention it, I do remember wondering whether the blockcache package should move over to the contrib. I'll do that